prototype/p2p/codec.go

```diff
@@ -140,6 +140,31 @@ func (c *wcodec) decode(wiredata []byte) (nonce uint32, enc Encoding, data []byt
 	return
 }
 
+func (c *wcodec) decodeHeader(header []byte) (flagswitch []bool, nonce uint32, bodylength uint32, err error) {
```

Comment:
`s/bodylength/bodyLength`

I'm pretty sure golang best practices is camelCase.

Ditto for `s/wcodec/wCodec`